packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -577,6 +583,16 @@ export class LoggingError extends Error implements ILoggingError {
     }
 }
 
+export class SomeLoggingError extends LoggingError {
```

Comment:
I do not know what "Some" means in this context.

Reply:
This was a placeholder name I forgot to fix up.  I need to ponder your comment above about making LoggingError abstract.  If I revert that part, then this class is no longer needed.  Might be unrelated to the main point of the PR and can be reverted anyway to be considered separately.

Reply:
Yep, reverting this.